README.md

```diff
@@ -95,13 +96,34 @@ NV_GPU='0' nvidia-docker run -it \
 
 It can also be helpful to mount the local copy of the repository and the scripts to speed up local development.
 
+However, if you are going to mount a local version of the repository, you must first run the docker image without the local repository mounted so that the c extensions can be compiled and then copied over to your local version.
+
 ```bash
+# First run the docker image without mounting externally
+NV_GPU='0' nvidia-docker run -it \
+  -p 8888:8888 \
+  $USER/deepcell-tf:latest
+  
+# Use ctrl-p, ctrl-q to exit the running docker image without shutting it down
+
+# Then, get the container_id after running the below command
+docker ps 
+
+# Replace the path/to/deepcell-tf below with the path to your local copy of deepcell-tf, and replace container_id with the container_id from the docker ps command above
```

Comment:
Can you make these into relative variables so users don't have to copy/paste the paths?  maybe you can name the docker container and reference it that way.

I would prefer this block of code (or any block of code) to be copy/pastable.

Reply:
Good call!